docs/contributor/storage/default_csi_plugin.md

```diff
@@ -275,4 +275,18 @@ Once the new pod enters the Running state, verify that the data we wrote early w
 ```shell
 oc exec base -- cat /data/demo.txt
 FOOBAR
+```
+
+# LVMS Versioning
+
+LVMS is released at a different cadence that MicroShift and is not couple to the MicroShift version.  This is primarily
+because LVMS is a subcomponent of MicroShift and deployed as a workload on the cluster.  The version of LVMS is tracked
+by image tag, with only the major version correlating the major MicroShift version.
+
+The LVMS version is not exposed by LVMS itself. For troublshooting purposes, MicroShift exposes the LVMS version 
+via a configmap in the `openshift-storage` namespace. To get the LVMS version, run the following command:
+
+```shell
+$ oc get configmap -n openshift-storage lvms-version -o jsonpath='{.data.version}'
+v4.14.0-10
```

Comment:
I think we want to display LVMS version when running `microshift version` as well?

Reply:
No, we don't want the CSI driver more tightly coupled to the core service, any more than we do for other add-ons.